packages/sdk-v2/src/sdk/newsfeed/sdk.ts

```diff
@@ -0,0 +1,163 @@
+import { assign, countBy, filter, get, groupBy, isArray, keys, last, map } from "lodash";
+
+import { CeramicModel as Post } from "./client";
+import { batch, isValidCID, serializeCollection, serializeDocument } from "./utils";
+import createIpfsStorage from "../ipfs/sdk";
+
+// move configs to constructor
+const ceramicBatchSize = 5;
+
+export interface FeedPost {
+  id: string;
+  link: string;
+  title: string;
+  content: string;
+  picture: string;
+  published: string;
+  sponsored_link: string;
+  sponsored_logo: string;
+}
+
+interface DocumentOrFeed {
+  document: FeedPost[];
+  picture: any;
+}
+
+export class CeramicFeed {
+  Post: Post;
+  IPFS: any;
+  constructor(ceramicConfig, ipfsUrls) {
```

Comment:
also, should receive instance of IPFS instead of ipfs url + instantiate ipfs here

Reply:
not relevant for orbis